editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -340,17 +344,22 @@ export function renderCoreElement(
     case 'JSX_CONDITIONAL_EXPRESSION': {
       const commentFlag = findUtopiaCommentFlag(element.comments, 'conditional')
       const override = isUtopiaCommentFlagConditional(commentFlag) ? commentFlag.value : null
-      const conditionValue: boolean =
+      const conditionValue =
```

Comment:
I think this explicit typing was required because the return type of `jsxAttributeToValue` is `any`. Maybe it's worth adding a runtime check that the value is of type `boolean`?

Reply:
I removed it because it would be valid in JavaScript to write something like `"definitelytrue" ? 1 : 2`, but treating that string as a boolean would definitely be wrong and conceivably it could end up being passed into some of our logic at some point and then all kinds of havoc could be wrought.

Reply:
To make our truthiness check happy I think I'll check if it is truthy then use that resulting boolean in the logic after that.

Reply:
Fixed in 60dbbfb90.